packages/backend/server/src/plugins/copilot/providers/openai.ts

```diff
@@ -292,37 +292,51 @@ export class OpenAIProvider
 
       const parser = new CitationParser();
       let lastType;
+      let prefix: string | null = '> [!]\n> ';
```

Comment:
is this the specific syntax for call out?
need a comments for this

Reply:
Ok, I added comments and constants